src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/GetNativeVariantForObjectTests.cs

```diff
@@ -115,28 +50,12 @@ public static IEnumerable<object[]> GetNativeVariantForObject_NonRoundtrippingPr
 
             // Wrappers.
             yield return new object[] { new UnknownWrapper(10), VarEnum.VT_UNKNOWN, IntPtr.Zero, null };
-            if (!PlatformDetection.IsNetCore)
-            {
-                yield return new object[] { new DispatchWrapper(10), VarEnum.VT_DISPATCH, IntPtr.Zero, null };
-            }
-            else
-            {
-                Assert.Throws<PlatformNotSupportedException>(() => new DispatchWrapper(10));
-            }
             yield return new object[] { new ErrorWrapper(10), VarEnum.VT_ERROR, (IntPtr)10, 10 };
             yield return new object[] { new CurrencyWrapper(10), VarEnum.VT_CY, (IntPtr)100000, 10m };
             yield return new object[] { new BStrWrapper("a"), VarEnum.VT_BSTR, (IntPtr)(-1), "a" };
             yield return new object[] { new BStrWrapper(null), VarEnum.VT_BSTR, IntPtr.Zero, null };
 
             yield return new object[] { new UnknownWrapper[] { new UnknownWrapper(null), new UnknownWrapper(10) }, (VarEnum)8205, (IntPtr)(-1), new object[] { null, 10 }  };
-            if (!PlatformDetection.IsNetCore)
-            {
-                yield return new object[] { new DispatchWrapper[] { new DispatchWrapper(null), new DispatchWrapper(10) }, (VarEnum)8201, (IntPtr)(-1), new object[] { null, 10 } };
-            }
-            else
-            {
-                Assert.Throws<PlatformNotSupportedException>(() => new DispatchWrapper(10));
```

Comment:
Creating a `DispatchWrapper` should no longer throw `PlatformNotSupportedException` on Windows in .NET Core, so I think the two cases currently under `!PlatformDetectior.IsNetCore` should be valid for Windows now?

Reply:
And it looks like the only test that uses this is scoped to Windows.

Reply:
It is valid, but not for `Int32`. I would need to find some type that supports IDispatch first.

Reply:
`IDispatchComObject`?

Reply:
Oh, wait, it looks there is `GetNativeVariantForObject_WrappedComObject_Success` for Windows.